src/linear_combination.jl

```diff
@@ -0,0 +1,59 @@
+"""
+    LinearCombination <: Transform
```

Comment:
```suggestion
    LinearCombination(coefficients) <: Transform
```

or just `coeffs`, whichever you think